src/jbi/whiteboard_actions/README.md

```diff
@@ -0,0 +1,21 @@
+## Looking to add a new action?
+"actions" are python functions that, once added into `src.jbi.actions.action_list`,
+can be run by updating the `action` attribute of the json config file.
```

Comment:
`Actions are Python modules, whose ``def init(payload, context)`` function is called when the Bugzilla Webhook endpoint is called.`

...not sure how to phrase it clearly, but this doc is outdated with your latest code ;)